Python/ast.c

```diff
@@ -9,6 +9,7 @@
 #include "ast.h"
 #include "token.h"
 #include "pythonrun.h"
+#include "internal/pystate.h"
```

Comment:
I think this is the culprit.

We probably need to undef Yield before and after it:
```suggestion
#undef Yield
#include "internal/pystate.h"
#undef Yield
```

And then at approx line 2680 change the use of `Yield(...)` to `_Py_Yield(...)`.

Reply:
Wow. I'm kind of amazed that compiled and linked at all rather than just emitted a compiler warning. Bad luck in terms of what the other macro expanded to I supposed. This fix appears to be working, the bot is able to produce an executable interpreter instead of one that calls an unknown Windows API during parsing now.